src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -134,7 +134,51 @@ public void SendKeys_Default_IsSupported()
 
             element.SendKeys("Hello World!");
 
-            Assert.That(element.Text, Is.EqualTo("Hello World!"));
+            Assert.That(element.Text, Is.EqualTo("Hello World!Test TextBox"));
+        }
+
+        [Test]
+        public void SendKeys_ShiftedCharacter_ShiftIs_Released()
+        {
+            var driverOptions = FlaUIDriverOptions.TestApp();
+            using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
+            var element = driver.FindElement(ExtendedBy.AccessibilityId("TextBox"));
+
+            element.SendKeys("!");
+            element.SendKeys("1");
+
+            Assert.That(element.Text, Is.EqualTo("!1Test TextBox"));
+        }
+
+        [Test]
+        public void SendKeys_DownArrow_IsSupported()
+        {
+            var driverOptions = FlaUIDriverOptions.TestApp();
+            using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
+            var element = driver.FindElement(ExtendedBy.AccessibilityId("NonEditableCombo"));
+
+            element.SendKeys(Keys.Down);
+
+            Assert.That(element.Text, Is.EqualTo("Item 2"));
+        }
+
+        [Test, Ignore("Alt key combinations currently fail due to https://github.com/FlaUI/FlaUI/issues/320")]
+        public void SendKeys_AltDownArrowEscape_IsSupported()
+        {
+            var driverOptions = FlaUIDriverOptions.TestApp();
+            using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
+            var element = driver.FindElement(ExtendedBy.AccessibilityId("NonEditableCombo"));
+            var expandCollapseState = element.GetDomAttribute("ExpandCollapse.ExpandCollapseState");
+
+            Assert.That(expandCollapseState, Is.EqualTo("Collapsed"));
+
+            element.SendKeys(Keys.Alt + Keys.Down);
+
+            Assert.That(expandCollapseState, Is.EqualTo("Expanded"));
+
+            Thread.Sleep(2000);
```

Comment:
I would rather end the test without checking whether the combo collapses again, to avoid this delay...

Reply:
Ah apologies, the sleep was there for debugging. I can remove that.